setup.cfg

```diff
@@ -87,3 +87,6 @@ ignore =
 [tool:pytest]
 addopts = --black --isort --pydocstyle --doctest-glob="*.rst" --doctest-modules --cov=invenio_theme --cov-report=term-missing tests invenio_theme
 testpaths = tests invenio_theme
+filterwarnings =
```

Comment:
I am not sure about this, I think we would like to know about deprecation warnings, even if it is only pytest

Reply:
normally I would say the same. In this special case there are already precedents like : [setup.cfg of invenio-i18n](https://github.com/inveniosoftware/invenio-i18n/blob/be68607e702cb34638f6dd07fda3acab365fe644/setup.cfg#L93). For me the most important argument for this change is that we can't fix those deprecation warnings, and we have so see them every time a test run which is annoying.

Reply:
removed